README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
(`BlobDeletionPillow` should be fixed by adding `SHARED_DRIVE_ROOT = '/some/valid/path/on/your/machine'` to `localsettings`)